libpod/container_exec.go

```diff
@@ -251,6 +256,20 @@ func (c *Container) ExecCreate(config *ExecConfig) (string, error) {
 	return session.Id, nil
 }
 
+func getPidData(pid int) string {
```

Comment:
Please add a comment here that there is a still race in generating the PIDData on start but this is as good as we can do currently, long term we should look into conmon sending us a pidfd back directly.